extensions/target-specific/stl-helpers/stl-helpers.js

```diff
@@ -53,6 +67,25 @@ DbgObject.AddTypeDescription(
     }
 );
 
+DbgObject.AddTypeDescription(
+    (type) => {
+        return type.name().match(/^std::__Cr::vector<(.*)>$/) != null;
+    },
+    "Size",
+    false,
+    (vector) => {
+        return Promise.all([vector.f("__begin_"), vector.f("__end_"), vector.f("__begin_").size()])
+        .thenAll((firstElement, lastElement, elementSize) => {
+            if (!firstElement.isNull()) {
```

Comment:
Consider sharing the code inside .thenAll between this and std::vector's Size type description?

Reply:
Will do before merging

Reply:
Done